packages/shared/src/components/modals/squads/PostModerationModal.tsx

```diff
@@ -85,16 +86,14 @@ function PostModerationModal({
             Post preview
           </Typography>
           {!isModerator && (
-            <div className="mr-4">
-              <SquadModerationItemContextMenu
-                id={id}
-                onDelete={onDeleteClick}
-              />
-            </div>
+            <SquadModerationItemContextMenu id={id} onDelete={onDeleteClick} />
           )}
-          <ModalClose onClick={modalProps.onRequestClose} />
+          <ModalClose
+            className="relative !right-0"
```

Comment:
Same here

Reply:
ModalClose actually accepts props to adjust right- :)

Reply:
Nice! Didn't see before, thanks for suggesting! 🙏